move/seal/Move.lock

```diff
@@ -0,0 +1,40 @@
+# @generated by Move, please check-in and do not edit manually.
+
+[move]
+version = 3
+manifest_digest = "2E00245C5FCD8C8921BA41FD38C270139216225A6D032B167C18DE701096800E"
+deps_digest = "F8BBB0CCB2491CA29A3DF03D6F92277A4F3574266507ACD77214D37ECA3F3082"
+dependencies = [
+  { id = "Sui", name = "Sui" },
+]
+
+[[move.package]]
+id = "MoveStdlib"
+source = { git = "https://github.com/MystenLabs/sui.git", rev = "testnet-v1.52.1", subdir = "crates/sui-framework/packages/move-stdlib" }
+
+[[move.package]]
+id = "Sui"
+source = { git = "https://github.com/MystenLabs/sui.git", rev = "testnet-v1.52.1", subdir = "crates/sui-framework/packages/sui-framework" }
+
+dependencies = [
+  { id = "MoveStdlib", name = "MoveStdlib" },
+]
+
+[move.toolchain-version]
+compiler-version = "1.59.0"
+edition = "2024.beta"
+flavor = "sui"
+
+[env]
+
+[env.testnet]
+chain-id = "4c78adac"
+original-published-id = "0x4614e5da0136ee7d464992ddd3719d388ae2bfdb48dfec6d9ad579f87341f2e1"
+latest-published-id = "0x4614e5da0136ee7d464992ddd3719d388ae2bfdb48dfec6d9ad579f87341f2e1"
```

Comment:
the one in the doc https://seal-docs.wal.app/UsingSeal/#on-chain-decryption is 0x927a54e9ae803f82ebf480136a9bcff45101ccbe28b13f433c89f5181069d682

Reply:
for original-published-id and latest-published-id

Reply:
I know the docs say `0x927a54e9ae803f82ebf480136a9bcff45101ccbe28b13f433c89f5181069d682` but I tried to deploy a contract that depended on that and I got errors with this package id `0x4614e5da0136ee7d464992ddd3719d388ae2bfdb48dfec6d9ad579f87341f2e1` instead. After updating as is I managed to deploy to testnet.

Although I had a different original-published-id, since [the explorer](https://suiscan.xyz/testnet/object/0x927a54e9ae803f82ebf480136a9bcff45101ccbe28b13f433c89f5181069d682/tx-blocks) says this is version 2